steps/fortify-on-demand-mobile-assessment/0.1.0/step.yml

```diff
@@ -0,0 +1,117 @@
+source_code_url: https://github.com/jeremy-chua/bitrise-step-fortify-on-demand-mobile-assessment
+support_url: https://github.com/jeremy-chua/bitrise-step-fortify-on-demand-mobile-assessment/issues
+published_at: 2021-10-20T08:27:18.204432733+08:00
+source:
+  git: https://github.com/jeremy-chua/bitrise-step-fortify-on-demand-mobile-assessment.git
+  commit: 91a5fb66253c1a3130ba6a4a7f981de85107480c
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- ios
+- android
+type_tags:
+- test
+toolkit:
+  go:
+    package_name: github.com/jeremy-chua/bitrise-step-fortify-on-demand-mobile-assessment
+is_requires_admin_user: false
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- client_id: null
+  opts:
+    description: Authenticating with Client Credentials. The client credentials are
+      the API keys generated in the portal.
+    is_expand: true
+    is_required: true
+    is_sensitive: true
+    summary: Client ID for authentication
+    title: Client ID
+- client_secret: null
+  opts:
+    description: Authenticating with Client Credentials. The client credentials are
+      the API keys generated in the portal.
+    is_expand: true
+    is_required: true
+    is_sensitive: true
+    summary: Client secret for authentication
+    title: Client Secret
+- datacenter: ' '
+  opts:
+    description: Select the datacenter assigned to you. This is provided once your
+      are onboarded with Fortify on Demand.
+    is required: true
+    is_expand: true
+    summary: Fortify on Demand Datacenters
+    title: Datacenter
+    value_options:
+    - AMS
+    - APAC
+    - EMEA
+    - FED
+- entitlement_id: null
+  opts:
+    description: The ID assigned to ypur current entitlement. This value can be reterived
+      in Foritfy on Demand portal.
+    is_expand: true
+    is_required: true
+    summary: Entitlement ID for this mobile assessment
+    title: Entitlement ID
+- opts:
+    description: The ID assigned to the created release. This value can be reterived
+      in Foritfy on Demand portal.
+    is_expand: true
+    is_required: true
+    summary: Release ID for this mobile assessment
+    title: Release ID
+  release_id: null
+- assessment_type: ' '
+  opts:
+    description: "Mobile Assessment includes Vulnerability analysis of mobile binary,
+      Endpoint reputation analysis and Security expert review of prioritized results.
+      \nMobile+ Assessment includes Manual testing of binary, network and services,
+      and DAST of backend services\nFor more information, please refer to the datasheet
+      at https://www.microfocus.com/media/data-sheet/fortify_on_demand_ds.pdf\n"
+    is required: true
+    is_expand: true
+    summary: Assessment type for this mobile assessment
+    title: Assessment Type
+    value_options:
+    - Mobile Assessment (Single Scan)
+    - Mobile+ Assessment (Single Scan)
+    - Mobile Assessment (Subscription)
+    - Mobile+ Assessment (Subscription)
+- framework_type: ' '
```

Comment:
Default value must be among `value_options`.

Reply:
I did not make any changes in step.yml
It was approved in previous versions 0.0.1 and 0.0.2
You can refer to 
https://github.com/bitrise-io/bitrise-steplib/pull/3225/files/9be8fbbd2d811f389e5243f4b9aa6b677ff51ef8